app/src/main/java/com/framgia/wsm/data/source/RequestRepository.java

```diff
@@ -73,20 +71,20 @@ public RequestRepository(RequestRemoteDataSource remoteDataSource) {
         return Observable.just(baseResponse);
     }
 
-    public Observable<BaseResponse<List<Request>>> getListRequestOverTime(@NonNull int userId) {
+    public Observable<BaseResponse<List<RequestOverTime>>> getListRequestOverTime(
+            @NonNull int userId) {
         // TODO: Edit later
         // return mRemoteDataSource.getListRequestOverTime(userId);
-        List<Request> requests = new ArrayList<>();
+        List<RequestOverTime> requestOverTimes = new ArrayList<>();
         for (int i = 0; i < 5; i++) {
-            Request request = new Request();
-            request.setCreatedAt("16/06/2017");
-            request.setStatus(2);
-            request.setFromTime("21/06/2017");
-            request.setToTime("22/06/2017");
-            requests.add(request);
+            RequestOverTime requestOverTime = new RequestOverTime();
+            requestOverTime.setFromTime("21/06/2017");
+            requestOverTime.setToTime("22/06/2017");
+            requestOverTimes.add(requestOverTime);
         }
-        BaseResponse<List<Request>> baseResponse = new BaseResponse<List<Request>>(requests) {
-        };
+        BaseResponse<List<RequestOverTime>> baseResponse =
+                new BaseResponse<List<RequestOverTime>>(requestOverTimes) {
+                };
```

Comment:
ERROR: 'object def rcurly' have incorrect indentation level 16, expected level should be one of the following: 8, 12.